docs/deployment_practices.md

```diff
@@ -91,6 +101,20 @@ Here's an Apache example that would exclude the landing page from logging:
     SetEnvIf Request_URI "^/securedrop$" dontlog
     CustomLog logs/access_log common env=!dontlog
 
+**Security suggestions**
+
+To guard your landing page against being modified by an attacker and directing sources to a rogue SecureDrop instance, you will need good security practices applying to the machine where it is hosted. Whether it's a VPS in the cloud or dedicated server in your office, you should consider the following:
+
+*		Brute force login protection (see sshguard or fail2ban)
```

Comment:
I copied that spacing from the next section "Minimum requirements for the SecureDrop environment". will fix both tho